bundles/org.openhab.persistence.influxdb/pom.xml

```diff
@@ -15,33 +15,124 @@
   <name>openHAB Add-ons :: Bundles :: Persistence Service :: InfluxDB</name>
 
   <properties>
-    <bnd.importpackage>!android.*,!com.android.*,!com.google.appengine.*,!org.apache.harmony.*,!org.apache.http.*,!rx.*</bnd.importpackage>
+    <bnd.importpackage>
+      !javax.annotation;!android.*,!com.android.*,!com.google.appengine.*,!dalvik.system,!kotlin.*,!kotlinx.*,!org.conscrypt,!sun.security.ssl,!org.apache.harmony.*,!org.apache.http.*,!rx.*,!org.msgpack.*
+    </bnd.importpackage>
   </properties>
 
   <dependencies>
-    <!-- https://mvnrepository.com/artifact/org.influxdb/influxdb-java -->
+    <!-- START InfluxDB 2.0 -->
+    <!-- START influxdb-client-java -->
     <dependency>
-      <groupId>org.influxdb</groupId>
-      <artifactId>influxdb-java</artifactId>
-      <version>2.2</version>
+      <groupId>com.influxdb</groupId>
+      <artifactId>influxdb-client-java</artifactId>
+      <version>1.6.0</version>
     </dependency>
-    <!-- https://mvnrepository.com/artifact/com.squareup.okhttp/okhttp -->
     <dependency>
-      <groupId>com.squareup.okhttp</groupId>
-      <artifactId>okhttp</artifactId>
-      <version>2.4.0</version>
+      <artifactId>influxdb-client-core</artifactId>
+      <groupId>com.influxdb</groupId>
+      <version>1.6.0</version>
+    </dependency>
+    <dependency>
+      <artifactId>converter-gson</artifactId>
+      <groupId>com.squareup.retrofit2</groupId>
+      <version>2.5.0</version>
+    </dependency>
+    <dependency>
+      <artifactId>converter-scalars</artifactId>
+      <groupId>com.squareup.retrofit2</groupId>
+      <version>2.5.0</version>
+    </dependency>
+    <dependency>
+      <artifactId>gson</artifactId>
+      <groupId>com.google.code.gson</groupId>
+      <version>2.8.5</version>
+    </dependency>
+    <dependency>
+      <artifactId>gson-fire</artifactId>
+      <groupId>io.gsonfire</groupId>
+      <version>1.8.0</version>
     </dependency>
-    <!-- https://mvnrepository.com/artifact/com.squareup.okio/okio -->
     <dependency>
-      <groupId>com.squareup.okio</groupId>
       <artifactId>okio</artifactId>
-      <version>1.4.0</version>
+      <groupId>com.squareup.okio</groupId>
+      <version>1.17.3</version>
+    </dependency>
+    <dependency>
+      <artifactId>commons-csv</artifactId>
+      <groupId>org.apache.commons</groupId>
+      <version>1.6</version>
+    </dependency>
+    <dependency>
+      <artifactId>json</artifactId>
+      <groupId>org.json</groupId>
+      <version>20180813</version>
+    </dependency>
```

Comment:
what needs this?

Reply:
It's used by 2.0 driver:
![image](https://user-images.githubusercontent.com/920260/78526581-b915a200-77da-11ea-8be7-b6f0305b22ba.png)